Hymns/Display/HymnLyricsView.swift

```diff
@@ -17,21 +17,29 @@ public struct HymnLyricsView: View {
         guard !lyrics.isEmpty else {
             return AnyView(Text("loading..."))
         }
+        var verseIndex: Int = 0
 
+        func isVerse(verseType: VerseType) -> Int {
+            if verseType != VerseType(rawValue: "chorus") {
+                verseIndex += 1
+                return verseIndex
+            }
+            return 0
+        }
         return AnyView(
-            ScrollView {
-                VStack(alignment: .leading) {
+            VStack(alignment: .leading) {
+                ForEach(lyrics, id: \.self) { verse in
                     Group {
-                        ForEach(lyrics, id: \.self) { verse in
-                            Group {
-                                ForEach(verse.verseContent, id: \.self) { line in
-                                    Text(line)
-                                }
-                                Spacer().frame(height: 30)
-                            }
+                        if verse.verseType == VerseType(rawValue: "chorus") {
```

Comment:
What's the difference between doing Group{ ForEach {...} } and doing ForEach{ Group{...} }?

Reply:
Uhh... I think in the master branch it was Group { ForEach { Group { ForEach } } }. I just removed one set of Group-ForEach. Otherwise I don't think there's much of a difference.

Reply:
Is ScrollView needed? I can keep that in there if so

Reply:
I think so, but try running and if it still scrolls fine then we can get rid of it

Reply:
It scrolls and builds fine on my local without ScrollView but I kept it in there just in case. Please review and i can merge to master.